src/core/Delta.ts

```diff
@@ -31,32 +31,103 @@ export enum DeltaQueue {
   kUnknown
 }
 
+/**
+ * Output JSON format for delta.json and deltaLog.json based on feedback
+ * from the AWG on 8/22/2023 to keep the output simple
+ * 
+ * So internally, we are storing the full CveCorePlus, but externally,
+ * and specifically when writing out to JSON, we are using this shortened format
+ 
+ * see https://github.com/CVEProject/cvelistV5/issues/23 for some additional discussions
+ * before and after the AWG meeting on 8/22
+ */
+export class DeltaOutpuItem {
+
+  static _cveOrgPrefix = `https://www.cve.org/CVERecord?id=`;
+  static _githubRawJsonPrefix = `https://raw.githubusercontent.com/CVEProject/cvelistV5/main/cves/`
+
+  cveId: string;          // string version of the CVE ID
+  cveOrgLink?: string;    // url to cve.org record
+  githubLink?: string;    // url to Github raw json
+  dateUpdated?: string;   // ISO string
+
+  static fromCveCorePlus(cvep: CveCorePlus): DeltaOutpuItem {
+    let deltaItem = new DeltaOutpuItem();
+    const cveid = cvep.cveId.toString();
+    deltaItem.cveId = cveid;
+    deltaItem.cveOrgLink = `${DeltaOutpuItem._cveOrgPrefix}${cveid}`;
+    deltaItem.githubLink = `${DeltaOutpuItem._githubRawJsonPrefix}${CveId.getCveDir(cveid)}/${cveid}.json`;
+    deltaItem.dateUpdated = cvep.dateUpdated;
+    return deltaItem;
+  }
+
+  static replacer(key: string, value: any) {
+    let items = [];
+    if (key === 'new' || key === 'updated' || key === 'unknown') {
+      value.forEach((item: CveCorePlus) => {
+        // Note, it is important to keep this loop as simple as possible,
+        //  don't rely on item being an actual CveCorePlus object since
+        //  it may not be depending on what built it
+        // console.log(`replacer item=${JSON.stringify(item, null, 2)}`);
+        const cveid = item.cveId.toString();
+        if (cveid) {
+          items.push({
+            cveId: cveid,
+            cveOrgLink: `${DeltaOutpuItem._cveOrgPrefix}${cveid}`,
+            githubLink: `${DeltaOutpuItem._githubRawJsonPrefix}${CveId.getCveDir(cveid)}/${cveid}.json`,
+            dateUpdated: item.dateUpdated
+          });
+        }
+      });
+      return items;
+    }
+    else {
+      return value;
+    }
+  }
+
+  toJSON() {
+    return {
+      cveId: this.cveId,
+      cveOrgLink: this.cveOrgLink,
+      githubLink: this.githubLink,
+      dateUpdated: this.dateUpdated
+    };
+  }
+
+
+}
+
 
-export class Delta /*implements DeltaProps*/ {
+export class Delta {
+
+  fetchTime?: string;
+  durationInMsecs?: number;   // if not set, it means that it was not calculated
```

Comment:
What does this duration represent exactly? The amount of time elapsed between this delta entry and the previous one?

Reply:
It is the elapsed time between the start of the operation (the `fetchTime`) and when the delta and deltaLog files were written (the end of the operation).  It is intended to take into account that the calls to the CVE REST services API and subsequent processing takes a few seconds to complete.  So if a CVE is added/updated on the REST service during this elapsed time, to expect it to show up on the _next_ scheduled update run (around 5-15 minutes) from the `fetchTime`.

Another way to look at this is that the list of "new" and "updated" CVEs in a delta entry is accurate as of the `fetchTime` ± the `durationInMsecs`.

Reply:
That sounds reasonable.  I'll comment out the `durationInMsecs` in a new push tonight.